pagopa/io-app
# chore: [IOBP-106] Add business name in IDPay transaction item

## Short description
This PR fixes a problem with the IDPay initiative timeline and adds the business name in the transaction details, if available.

## List of changes proposed in this pull request
- Updated IDPay API specs in `package.json` to **v2.107.3**
- Updated `TimelineDiscountTransactionDetailsComponent` with `businessName`
- Updated `TimelineOperationListItem` with `businessName`

## How to test
With the `io-dev-server`, checkout this PR https://github.com/pagopa/io-dev-api-server/pull/279. Navigate to the details page of an initiative and check if the business name is correctly displayed.
